File: viewlaboratory/src/main/java/com/skx/tomike/viewlaboratory/activity/ShapeViewActivity.java
package com.skx.tomike.viewlaboratory.activity;

import android.os.Bundle;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;

import com.skx.tomike.viewlaboratory.R;
import com.skx.tomike.viewlaboratory.view.ScoreView;

import java.util.Random;

public class ShapeViewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_shape_view);

        ScoreView scoreView = findViewById(R.id.customView_scoreView);
        refreshScore(scoreView);
    }

    private void refreshScore(final ScoreView scoreView) {
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Random r = new Random();
                int i = r.nextInt(20);
                scoreView.setIndicatorPos(i);

                refreshScore(scoreView);
            }
        }, 2000);
    }
}

File: viewlaboratory/src/main/java/com/skx/tomike/viewlaboratory/activity/ShapeViewActivity.kt
package com.skx.tomike.viewlaboratory.activity

import android.os.Bundle
import android.os.Handler
import android.support.v7.app.AppCompatActivity
import com.skx.tomike.viewlaboratory.R
import com.skx.tomike.viewlaboratory.view.ScoreView
import java.util.*

class ShapeViewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_shape_view)

        val scoreView = findViewById<ScoreView>(R.id.customView_scoreView)
        refreshScore(scoreView)
    }

    private fun refreshScore(scoreView: ScoreView) {
        Handler().postDelayed({
            val r = Random()
            val i = r.nextInt(20)
            scoreView.setIndicatorPos(i)

            refreshScore(scoreView)
        }, 2000)
    }
}
